config/routes.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  root 'static#index'
+  namespace :v1, defaults: { format: :json } do
+    get 'messages', to: 'messages#index'
+  end
```

Comment:
- [ ] Currently, I'm not able to see all the messages after correcting and running the seed file. It's giving me the route error. Please fix this.

![Screenshot from 2022-02-24 00-54-12](https://user-images.githubusercontent.com/22434660/155395705-c21241d4-612b-4aef-a3e5-874e1f9eb6e8.png)